src/ImageSharp.Web.Providers.AWS/Caching/AWSS3StorageCache.cs

```diff
@@ -82,20 +83,23 @@ public Task SetAsync(string key, Stream stream, ImageCacheMetadata metadata)
     /// and object data. <see cref="S3CannedACL.Private"/> specifies that the bucket
     /// data is private to the account owner.
     /// </param>
+    /// <param name="serviceProvider">The current service provider.</param>
     /// <returns>
     /// If the bucket does not already exist, a <see cref="PutBucketResponse"/> describing the newly
     /// created bucket. If the container already exists, <see langword="null"/>.
     /// </returns>
     public static PutBucketResponse? CreateIfNotExists(
         AWSS3StorageCacheOptions options,
-        S3CannedACL acl)
-        => AsyncHelper.RunSync(() => CreateIfNotExistsAsync(options, acl));
+        S3CannedACL acl,
+        IServiceProvider serviceProvider)
```

Comment:
This change is a breaking change, because it requires users of this method to add the serviceProvider param.